File: bj/2292.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    int start = 1;
    int d = 1;
    cin >> n;
    while(true){
        if(n < start + 1){
            break;
        }
        else{
            start += 6*d;
            d += 1;
        }
    }
    cout << d;
    return 0;
}

File: bj/2292.py
import sys
n = int(sys.stdin.readline().strip())

def path_num(num):
    start = 1
    d = 1
    while True:
        if num < start + 1:
            return d
        else:
            start += 6*d
            d += 1

print(path_num(n))